demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -575,7 +584,8 @@ def is_valid_fromversion_for_generic_objects(self):
             return True
 
         if LooseVersion(
-            self.current_file.get("fromVersion", DEFAULT_CONTENT_ITEM_FROM_VERSION)
+            self.current_file.get(
```

Comment:
this seems to happen due to different `black` configs in your local env and pre-commit. Please run `pre-commit run -a` to be able to merge